test/test_exporter/test_consensus.py

```diff
@@ -194,6 +194,38 @@ def test_format_consensus_categorical_nested(self, user_info):
         res = format_consensus(rows)
         assert res[0]['consensus__context.2000.ny.name__contributorsConsensusPercentage'] == expect
 
+    @with_context
+    @patch('pybossa.exporter.consensus_exporter.get_user_info')
+    def test_format_consensus_empty_answer_field_config(self, user_info):
+        user_info.return_value = {'name': 'joe'}
+        consensus = {
+            "context.name": {
+                "answer_field_config": None,
+                "contributorsMetConsensus": [1],
+                "percentage": 100.0,
+                "contributorsConsensusPercentage": [{
+                    "percentage": 100.0,
+                    "user_id": 1
+                }],
+                "value": "hello"
```

Comment:
should the value be "hello" here?

Reply:
it is just the placement for the mock taskruns.